todo.js

```diff
@@ -7,17 +7,25 @@ const todoItemsList = document.querySelector('.todo-items')
 
 const totalTasks = document.querySelector(".total-tasks span")
 const completedTasks = document.querySelector(".completed-tasks span")
-const remainingTasks = document.querySelector(".remaining-tasks span")
+// const remainingTasks = document.querySelector(".remaining-tasks span")
 
 
+const currentDay=new Date()
+const showcurrentDay=currentDay.toLocaleString('default',{weekday:'long'})
+document.getElementById('day').innerHTML=showcurrentDay
+
+const currentDate=new Date()
```

Comment:
no need to initialize this variable you already have a date object on top, reuse that variable to get the current date